python/ThirteenTeV/MonoZ/dmsimp_spin0/MonoZ_DMSimp_Spin0_cff.py

```diff
@@ -0,0 +1,45 @@
+import FWCore.ParameterSet.Config as cms
+
+externalLHEProducer = cms.EDProducer("ExternalLHEProducer",
+    args = cms.vstring(''),
```

Comment:
this is only a template, right?

Reply:
Yes exactly. It's the same hadronizer for both points.